docs/types/Store.prototype.createData.md

```diff
@@ -3,8 +3,38 @@
 
 @signature `Store.prototype.createData(request, response)`
 
-A `requestHandler` that creates an item in the store.
+  A `requestHandler` that creates an item in the store.
 
-```js
-fixture( "POST /api/todos", todoStore.createData );
-```
+  ```js
```

Comment:
There's no introduction to this example.  

Also, at some point, it should be mentioned that normally `store.createData` is wired to a url either through something like: `fixture("/api/todos/{id}", store)` or `fixture("POST /api/todos", store.createData)`.